### PR TITLE
Update Go gRPC sample code

### PR DESCRIPTION
**Pull request by jyap808**
Update Go gRPC sample code

### CODE REVIEW

File: rpc/documentation/clientusage.md [hunk 1]
```diff
@@ -60,14 +63,32 @@ import (
...
 func main() {
-	creds, err := credentials.NewClientTLSFromFile(certificateFile, "localhost")
+	serverCAs := x509.NewCertPool()
+	serverCert, err := ioutil.ReadFile(certificateFile)
+	if err != nil {
+		fmt.Println(err)
+		return
+	}
+	if !serverCAs.AppendCertsFromPEM(serverCert) {
+		fmt.Println(err)
```
Comment (jrick) [inline rpc/documentation/clientusage.md:80]: this should print some informative message instead of the err variable, which will always be nil here

**Review by jrick — CHANGES_REQUESTED** [review]
thanks! I guess all of the other examples should get some updates as well for the required auth, but i won't hold up this pr for that improvement.

just a small nit below, otherwise this looks good.

File: rpc/documentation/clientusage.md (hunk 1 above)
Comment (jyap808) [reply rpc/documentation/clientusage.md]: Resolved with https://github.com/decred/dcrwallet/pull/1907/commits/dcda70caa46a99108b838b8cab598265bec60814

File: rpc/documentation/clientusage.md [hunk 2]
```diff
@@ -60,14 +63,32 @@ import (
...
 func main() {
-	creds, err := credentials.NewClientTLSFromFile(certificateFile, "localhost")
+	serverCAs := x509.NewCertPool()
+	serverCert, err := ioutil.ReadFile(certificateFile)
+	if err != nil {
+		fmt.Println(err)
+		return
+	}
+	if !serverCAs.AppendCertsFromPEM(serverCert) {
+		fmt.Println("AppendCertsFromPEM error")
```
Comment (davecgh) [inline rpc/documentation/clientusage.md:80]: I would suggest:

```Go
fmt.Printf("no certificates found in %s\n", certificateFile)
```

That is really what the issue is here and I think is a much more obvious error from a user perspective.

**Review by davecgh — COMMENTED** [review]
Looks good overall.  I added a suggested output message in the case of the certificate file not having any certificates.

File: rpc/documentation/clientusage.md (hunk 2 above)
Comment (jyap808) [reply rpc/documentation/clientusage.md]: Resolved with https://github.com/decred/dcrwallet/pull/1907/commits/5bf0244831151ef4e9a5dd09786d28e9feb21242

**Review by davecgh — APPROVED** [review]

**Review by jrick — APPROVED** [review]

**Pull request merged**